BUILD.bazel

```diff
@@ -472,41 +472,254 @@ ray_cc_library(
 
 ray_cc_library(
     name = "plasma_store_server_lib",
+    srcs = ["src/ray/object_manager/plasma/store_runner.cc"],
+    hdrs = ["src/ray/object_manager/plasma/store_runner.h"],
+    copts = PLASMA_COPTS,
+    linkopts = PLASMA_LINKOPTS,
+    deps = [
+        ":object_lifecycle_manager",
+        ":object_store",
+        ":plasma_allocator",
+        ":store",
+        "//src/ray/common:asio",
+        "//src/ray/common:file_system_monitor",
+        "//src/ray/common:ray_config",
+        "//src/ray/util:thread_utils",
+        "@com_google_absl//absl/synchronization",
+    ],
+)
+
+ray_cc_library(
+    name = "store",
```

Comment:
btw, I quickly checked manually, unordered_map and unordered_set is not used anywhere, could you please do me a favor to remove them from inclusion?